ganga/GangaGUI/api/routes.py

```diff
@@ -530,6 +530,10 @@ def jobs_statistics():
         # Store statistics
         for stat in ["new", "running", "completed", "failed", "killed"]:
             statistics[stat] = len(jobs.select(status=stat))
+            if stat == "completed":
+                statistics[stat] = len(jobs.select(status="completed_frozen"))
+            elif stat == "failed":
+                statistics[stat] = len(jobs.select(status="failed_frozen"))
```

Comment:
Should these not be added to the numbers for completed and failed (rather than replacing them).

Reply:
yes